README.md

```diff
@@ -16,50 +24,52 @@ The crawl results can be output in 3 different modes:
 
 ### Setup
 
+In order to index crawl results into an Elasticsearch instance, you must first have one up and running.
+
 #### Running from Docker
 
 Crawler has a Dockerfile that can be built and run locally.
 
-1. Build the image `docker build -t crawler-image .`
-2. Run the container `docker run -i -d --name crawler crawler-image`
+1. Clone the repository
+2. Build the image `docker build -t crawler-image .`
+3. Run the container `docker run -i -d --name crawler crawler-image`
    - `-i` allows the container to stay alive so CLI commands can be executed inside it
    - `-d` allows the container to run "detached" so you don't have to dedicate a terminal window to it
-3. Confirm that Crawler commands are working `docker exec -it crawler bin/crawler version`
-4. Execute other CLI commands from outside of the container by prepending `docker exec -it crawler <command>`.
+4. Confirm that Crawler commands are working `docker exec -it crawler bin/crawler version`
+5. Execute other CLI commands from outside of the container by prepending `docker exec -it crawler <command>`.
    - See [Crawling content](#crawling-content) for examples.
 
 #### Running from source
```

Comment:
suggestion: As Docker is recommended option, we could clear up real estate on this page by [collapsing](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab) the Running from source instructions